pages/privacy.js

```diff
@@ -1,10 +1,12 @@
-import DynamicMarkdownContent from '../components/DynamicMarkdownContent'
+import { createGetStaticProps, createMarkdownContent } from '../components/createMarkdownContent'
 
-export default function Privacy () {
-  return (
-    <DynamicMarkdownContent
-      title='Privacy Policy'
-      url={process.env.PRIVACY_POLICY_URL}
-    />
-  )
-}
+/**
+ * getStaticProps function for statically embedding markdown.
+ * (see https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation).
+ */
+export const getStaticProps = createGetStaticProps(process.env.PRIVACY_POLICY_URL)
+
+/**
+ * Component that renders the privacy policy page.
+ */
+export default createMarkdownContent('Privacy Policy')
```

Comment:
I'd prefer that we stick to the standard `export default function Privacy ()` format for these pages. Abstracting too much away just makes things harder to follow. Same goes for `getStaticProps`. I'm fine with some abstraction (especially for the content fetching), but it gets confusing when things don't look the way they do in the docs, especially for folks that are less familiar with this stuff.

Reply:
I agree I pushed it a little too far.